<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarManager</class>
 <widget class="QMainWindow" name="CarManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>601</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>CarManager</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;:/images/background.jpg&quot;);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>1031</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>210</y>
          <width>221</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0.05);
border-radius: 12px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QPushButton" name="toggleDrivingModeButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>SF Pro Display</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 20px;
border-radius: 10px;
padding: 8px;
color: white;
background-color: rgba(255, 255, 255, 0.1);
border: 1px solid rgba(255, 255, 255, 0.2);</string>
         </property>
         <property name="text">
          <string>Toggle driving mode</string>
         </property>
        </widget>
        <widget class="QPushButton" name="toggleMetricsButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>SF Pro Display</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 20px;
border-radius: 10px;
padding: 8px;
color: white;
background-color: rgba(255, 255, 255, 0.1);
border: 1px solid rgba(255, 255, 255, 0.2);</string>
         </property>
         <property name="text">
          <string>Toggle metrics</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="inferenceLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>261</width>
          <height>151</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 16px;
border: 2px solid rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0.03);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="settingsLabel">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>170</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background: transparent;</string>
        </property>
        <property name="text">
         <string>Settings:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QToolButton" name="wifiToggleButton">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>0</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton {
    background: transparent;
    border: none;
    outline: none;
    padding: 0px;
}
QToolButton::menu-indicator {
    image: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/wifi.png</normaloff>:/images/wifi.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="timeLabel">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>0</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background: transparent;
</string>
        </property>
        <property name="text">
         <string>20:10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="dateLabel">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>100</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background: transparent;</string>
        </property>
        <property name="text">
         <string>Nov 20</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="drivingModeLabel">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>370</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background: transparent;
</string>
        </property>
        <property name="text">
         <string>Manual</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="drivingModeLabel_2">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>340</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background: transparent;
</string>
        </property>
        <property name="text">
         <string>Driving Mode:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="directionDriveLabel">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>160</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>26</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white; background: transparent;
</string>
        </property>
        <property name="text">
         <string>D</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="directionNeutralLabel">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>160</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>26</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background: transparent;
</string>
        </property>
        <property name="text">
         <string>N</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="directionReverseLabel">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>160</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>26</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white; background: transparent;
</string>
        </property>
        <property name="text">
         <string>R</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="directionParkLabel">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>160</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>26</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white; background: transparent;
</string>
        </property>
        <property name="text">
         <string>P</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="speedMetricsLabel">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>320</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background: transparent;
</string>
        </property>
        <property name="text">
         <string>KM/H</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="speedLabel">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>270</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>40</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background: transparent;
</string>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="batteryLabel">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>6</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background: transparent;
</string>
        </property>
        <property name="text">
         <string>100%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="temperatureLabel">
        <property name="geometry">
         <rect>
          <x>920</x>
          <y>6</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background: transparent;
</string>
        </property>
        <property name="text">
         <string>22.3 ºC</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="mileageLabel">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>430</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SF Pro Display</family>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background: transparent;
</string>
        </property>
        <property name="text">
         <string>10 m</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="batteryIconLabel">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>0</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/images/battery.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="temperatureIconLabel">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>6</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/images/temperature.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="laneKeepingAssistLabel">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>210</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/images/lane-keeping-off.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="speedLimit80Label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>130</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/images/speed-limit-80.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="speedLimit50Label">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>130</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/images/speed-limit-50.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
